File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count=0, res=0;
        for (int i:nums){
            count= i ? count+1:0;
            res=max(count, res);
        }
    return res;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numset=set(nums)
        res=0

        for i in nums:
            if i-1 not in numset:
                l=1
                while(i+l) in numset:
                    l+=1
                res= max(res,l)
        return res
        







        